madminer/utils/morphing.py

```diff
@@ -5,6 +5,8 @@
 from collections import OrderedDict
 from typing import Dict
 from typing import Iterable
+from sympy import Poly, symbols, expand
+from numpy import Infinity, linalg as la
```

Comment:
I would rather have Sympy being imported as `sp`, instead of importing specific functions / modules directly. Reason being: `symbols` and `expand` as generic enough words that may collide with variable / function names  in the future.

Not sure if there is any reason why this should not be the case, I am not familiar with this library.

Reply:
yes i think it can be changed to import as sp